Tanvir04Khan/01-max-udemy/03-login-page/src/App.js

```diff
@@ -1,17 +1,27 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 
-import Header from "./Components/Login/Header";
+import LoginPage from "./Components/Login/LoginPage";
+import HomePage from "./Components/Home/HomePage";
 
 import "./App.css";
 
 function App() {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
 
+  useEffect(() => {
+    if (localStorage.isLoggedIn) {
+      setIsLoggedIn(true);
+    }
+  }, []);
+
+  const authentication = (isUserLoggedIn) => {
```

Comment:
@Tanvir04khan  function name should start with verbs like:
`set....`
`handle....`
`get......`
`render.....`
`authenticate.....`
`check......`
`compute......`

Reply:
changed code as per suggestion @faran4engg